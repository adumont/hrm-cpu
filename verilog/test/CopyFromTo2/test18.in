21 3b
51 91
18 52
02 81
54 8d
23 52
20 e2
52 ee
40 ef
57 40
48 bc
41 75
12 25
18 16
31 08
27 e2
16 be
19 d2
54 a6
42 99
50 8d
48 19
06 d9
09 40
45 72
23 ad
01 57
01 33
17 45
19 d2
32 c3
15 b2
12 67
02 cd
54 8e
57 aa
58 f9
02 1a
13 02
30 6d
06 47
48 e3
46 e9
34 34
53 e1
51 21
19 54
