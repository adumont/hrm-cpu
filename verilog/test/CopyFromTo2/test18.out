3b
91
52
81
8d
52
e2
ee
ef
40
bc
75
25
16
08
e2
be
d2
a6
99
8d
19
d9
40
72
ad
57
33
45
d2
c3
b2
67
cd
8e
aa
f9
1a
02
6d
